De novo mRNAseq workshop
================================

.. @add mailing list info

Who: `Titus Brown <mailto:ctbrown@ucdavis.edu>`__ (lead instructor)

When: October 26 and 27, 2015

Times: 9am-4pm on both days

Where: 1043 Valley Hall, UC Davis campus

Cost: There is no fee. 

Host: C. Titus Brown and Jessica Mizzi; contact `Jessica Mizzi <mailto:jessica.mizzi@gmail.com>`__ with any questions.

This workshop is open to everyone, including graduate students,
postdocs, staff, faculty, and community members.  We have extra space
for UC Davis VetMed affiliates; contact the lead instructor if you are
an SVM affiliate.

`> Register here < <https://www.eventbrite.com/e/de-novo-mrna-seq-tickets-19046171612>`__
---------------------------------------------------------------------------------------------------------------

.. `> Materials link < <http://2015-may-nonmodel.readthedocs.org/en/latest/>`__
.. --------------------------------------------------------------------------


Description
-----------

This two-day hands-on workshop will introduce biologists to doing de
novo transcriptome analysis with khmer and Trinity.  The primary focus
will be on non-model organisms - organisms with no reference genome
and poor gene annotations.  The workshop will be eukaryote-focused
(poly-A mRNA/large genomes/genes with introns).

We will be analyzing a stock data set, and we will be using the Amazon
cloud.

Topics overview
~~~~~~~~~~~~~~~

* Non-model organisms and RNAseq: an overview of the options
* Logging into the Amazon Cloud
* Short read quality and trimming
* Building a new reference transcriptome with khmer and Trinity
* Data analysis & differential expression

You might also be interested in `the Eel Pond mRNAseq protocol
<https://khmer-protocols.readthedocs.org/en/latest/mrnaseq/index.html>`__,
which lays out a more general protocol for de novo mRNAseq assembly.

Schedule
~~~~~~~~

* Mon, 9am-noon: Amazon cloud, read trimming, basic sequence cleaning
* Mon, noon-1pm: lunch
* Mon, 1-4pm, assembly

* Tuesday, 9am-noon: annotation, mapping
* Tuesday, noon-1pm: lunch
* Tuesday, 1pm-4pm: quantification and differential expression analysis

What we covered last time
~~~~~~~~~~~~~~~~~~~~~~~~~
* `mRNAseq on non-model systems <http://2015-may-nonmodel.readthedocs.org/en/dev/>`__ (last updated: May 2015)


Computer requirements
~~~~~~~~~~~~~~~~~~~~~

Attendees will need to bring a computer with a Web browser, an
Internet connection, and an ssh client; Windows users should install
`MobaXterm <http://mobaxterm.mobatek.net/>`__ before the workshop.
