mRNAseq for semi-model organisms 
================================

.. @add mailing list info

Who: `Titus Brown <mailto:ctbrown@ucdavis.edu>`__ (lead instructor);
Dr. Isabelle Laforest-Lapointe; Dr. Tamer Mansour.

When: March 4 and 5, 2015

Times: 9am-3pm on both days

Where: Valley Hall, UC Davis campus.  Day 1: 2030 Valley; Day 2: 1041 Valley.

Cost: there is no fee.

This workshop is open to everyone, including graduate students,
postdocs, staff, faculty, and community members.  We have extra space
for UC Davis VetMed affiliates; contact the lead instructor if you are
an SVM affiliate.

.. (These spaces will be released to the wait list
on Thursday, February 26th.)

.. `> Register here < <https://www.eventbrite.com/e/mrnaseq-workshop-semi-model-organisms-registration-15830131349>`__
---------------------------------------------------------------------------------------------------------------

`> Materials link < <http://2015-mar-semimodel.readthedocs.org/en/latest/>`__
---------------------------------------------------------------------------

Description
-----------

This two-day hands-on workshop will introduce biologists to doing
reference-based transcriptome analysis with TopHat, Cufflinks, and
DAVID.  The primary focus will be on "semi-model" organisms -
organisms with good quality reference genomes but relatively
incomplete gene annotations.  This includes most animals and plants
that are not a major model organism.  The workshop will be
eukaryote-focused (large genomes, and genes with introns).

We will be analyzing a stock data set, and we will be using the Amazon
cloud.

Topics overview
~~~~~~~~~~~~~~~

* Semi-model organisms and RNAseq: an overview of the options
* Logging into the Amazon Cloud
* Short read quality and trimming
* Building a new reference transcriptome with TopHat and Cufflinks
* Mapping reads to the transcriptome with TopHat
* Processing another sample with TopHat and HTSeq
* Data analysis & differential expression

The materials will be based off of `these materials from a 2014 course
held at MSU
<http://2014-msu-rnaseq.readthedocs.org/en/latest/semi-orgs.html>`__,
and we will link to the updated materials from this site.  All materials
will be available indefinitely after the workshop.

Schedule
~~~~~~~~

* Wed, 9am-noon: Amazon cloud, read trimming, basic sequence cleaning
* Wed, noon-1pm: lunch
* Wed, 1-3pm, building gene models

* Thursday, 9am-noon: mapping, quantification
* Thursday, noon-1pm: lunch
* Thursday, 1pm-3pm: differential expression analysis

Computer and workshop requirements
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Attendees will need to bring a computer with a Web browser, an
Internet connection, and an ssh client; Windows users should install
`MobaXterm <http://mobaxterm.mobatek.net/>`__ before the workshop.

Please sign up for an account at http://aws.amazon.com/ before the workshop.
